packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -59,6 +60,8 @@ export type AccessRequirementListProps = {
   accessRequirementFromProps?: Array<AccessRequirement>
   /* If provided, displays these actions on when viewing all ARs */
   customDialogActions?: ReactNode
+  /* Optional callback invoked if a submission is created via this component */
+  onSubmissionCreated?: (submissionId: string) => void
```

Comment:
On the new request page, we will push a toast with a button to go to the page of the new submission.